server/src/main/java/io/crate/planner/node/ddl/AlterTableAddColumnPlan.java

```diff
@@ -115,20 +83,13 @@ public static AddColumnRequest createRequest(AnalyzedAlterTableAddColumn alterTa
                                                  Row params,
                                                  SubQueryResults subQueryResults,
                                                  FulltextAnalyzerResolver fulltextAnalyzerResolver) {
-        AnalyzedTableElements<Object> tableElements = validate(
+        var tableElements = validate(
             alterTable,
             plannerContext.transactionContext(),
             nodeContext,
             params,
-            subQueryResults
-        );
-
-        AnalyzedTableElements.validateAndBuildSettings(tableElements, fulltextAnalyzerResolver);
-
-        AnalyzedTableElements.finalizeAndValidate(
-            alterTable.tableInfo().ident(),
-            alterTable.analyzedTableElementsWithExpressions(),
```

Comment:
Moved those 2 method under validate and using it now in `AlterTableAddColumnAnalyzerTest`.  It used to validate implicitly via `bind` calls but now `bind` is gone so I'm using `validate` directly there.